Sources/ConsoleKit/Console.swift

```diff
@@ -47,6 +47,21 @@ public protocol Console: class {
     ///
     /// - parameters:
     ///     - secure: If `true`, the input should not be shown while it is entered.
+    ///
+    /// - returns: the string that was read, or `nil` if EOF was encountered
+    func read(isSecure: Bool) -> String?
+
+    /// Returns a `String` of input read from the `Console` until a line feed character was found.
+    ///
+    ///     let input = console.input(isSecure: false)
+    ///     print(input) // String
+    ///
+    /// - note: The line feed character should not be included.
+    ///
+    /// - parameters:
+    ///     - secure: If `true`, the input should not be shown while it is entered.
+    ///
+    /// - returns: the string that was read, or an empty string if EOF was encountered
     func input(isSecure: Bool) -> String
```

Comment:
Should this be deprecated?

Reply:
I actually marked it deprecated at first, but of course that results in the `console-kit` package itself spewing deprecation errors that users can't do anything about, which is a non-starter. Naturally, Swift still has no way to silence deprecation warnings for such a case despite many requests for it. I figure we can just fix it up in Vapor 5 anyhow.

Reply:
👍